sdk/servicebus/azure-servicebus/azure/servicebus/_common/receiver_mixins.py

```diff
@@ -20,6 +20,7 @@
 class ReceiverMixin(object):  # pylint: disable=too-many-instance-attributes
     def _populate_attributes(self, **kwargs):
         self._amqp_transport: Union["AmqpTransport", "AmqpTransportAsync"]
+        self.entity_path: str
```

Comment:
was this to remove Optional for entity_path ?

Reply:
It was, but the user should never be able to call this when it's set to None. We populate this value to a string (self._entity_name) in the constructor. We raise if this property is not populated: https://github.com/Azure/azure-sdk-for-python/blob/630b103822954c47fb535222ae6d4458775920d6/sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py#L192